### ISSUE TITLE
Using airflow images from Azure market place

### ISSUE THREAD

**Issue by maneeshmenon** [bitnami/bitnami-docker-airflow#85]
Hi,

I used the docker compose to successfully setup airflow

Documentation
- https://github.com/bitnami/bitnami-docker-airflow#readme
- https://raw.githubusercontent.com/bitnami/bitnami-docker-airflow/master/docker-compose.yml

Deployment works fine and I am able to login to http://ip_address:8080

Now I wish to replace the following three images with the images provide in Azure market place.

Original images
1. docker.io/bitnami/airflow-scheduler:2
2. docker.io/bitnami/airflow-worker:2
3. docker.io/bitnami/airflow:2

Azure images are explained here.
https://azuremarketplace.microsoft.com/en-us/marketplace/apps/bitnami.airflow-scheduler-container
https://azuremarketplace.microsoft.com/en-us/marketplace/apps/bitnami.airflow-worker-container
https://azuremarketplace.microsoft.com/en-us/marketplace/apps/bitnami.airflow-container

However, I fail to deploy, if I replace the orignal bitnami images with the images provided by Azure.

What could be the reason. How to debug?

Thanks and Regards,
Maneesh Menon

**Comment by maneeshmenon** [comment]
Hi,

If we use the docker-compose as explained in the link: https://github.com/bitnami/bitnami-docker-airflow, then following should be the initial logs from the airflow container.

airflow 10:26:55.50 Welcome to the Bitnami airflow container
airflow 10:26:55.50 Subscribe to project updates by watching https://github.com/bitnami/bitnami-docker-airflow
airflow 10:26:55.51 Submit issues and feature requests at https://github.com/bitnami/bitnami-docker-airflow/issues
airflow 10:26:55.52
airflow 10:26:55.54 INFO  ==> Enabling non-root system user with nss_wrapper
airflow 10:26:55.59 INFO  ==> ** Starting Airflow setup **
airflow 10:26:55.71 INFO  ==> Initializing Airflow ...
airflow 10:26:55.81 INFO  ==> No injected configuration file found. Creating default config file
airflow 10:26:57.56 INFO  ==> Configuring Airflow webserver authentication
airflow 10:26:57.60 INFO  ==> Configuring Airflow database
airflow 10:26:57.66 INFO  ==> Configuring Celery Executor
airflow 10:26:57.72 INFO  ==> Waiting for PostgreSQL to be available at postgresql:5432...
airflow 10:26:57.76 INFO  ==> Restoring persisted Airflow installation



When I use the Microsoft Azure Market place image for airflow, then what I observe is that airflow container exists immediately.

The following message is not part of the container log ::: "Enabling non-root system user with nss_wrapper"

Is this the issue?

Regards,
Maneesh Menon

**Comment by juan131** [comment]
Hi @maneeshmenon 

According to the logs you shared, you didn't clean the docker volumes attached to the containers when you replace the images from DockerHub with the ones taken from Azure Marketplace.

To discard the issue is related with the persistence, could you please ensure that you start the Airflow container on a "clean" environment? I mean, that you remove the volumes before starting:

- Stop the containers and remove volumes:

```console
$ docker-compose down -v

- Apply the changes in the Docker Compose to substitute DockerHub images with the ones from Azure Marketplace
- Launch the containers again:

```console
$ docker-compose up -v
```

Then, please let us know if you face any issue.

Best regards,

Juan Ariza
